lib/radio_utils/image_queue.py

```diff
@@ -0,0 +1,52 @@
+"""The Transmission Queue is a max heap of messages to be transmitted.
+
+Messages must support the `__lt__`, `__le__`, `__eq__`, `__ge__`, and `__gt__` operators.
+This enables to the max heap to compare messages based on their priority.
+"""
+from .queue import Queue
```

Comment:
you definitely just coppied this from the other queue, but this is probably a bad idea to add all this boiler plate just to make a slightly easier import.

At least on the pycubed-mini side we started having memory issues and stuff like this doesn't help.

Not at all your fault given this is how it's done else where. I'd either just make it the three lines needed ie:
```python
from .queue import Queue
limit = 100
image_queue = Queue(limit)
```
Or leave a issue somewhere to do this later (so we don't forget).

Reply:
I agree with Losha here, We could make this a much smaller file.